### DISCUSSION TITLE
How do I request an in-depth mars surface scan for my team?

### DISCUSSION THREAD

**Discussion by True-Texan** [SimDynamX/SpaceTeamsPRO#16] [2023 Mars City Design Competition Q&A] [answered]
I was looking through the introductory videos and I couldn't find the proper way to request a more precise surface scan of the area of the mars surface that my team wants to build on, but it's mentioned in the getting started flyer. How do I get that info?

**Comment by connorjak** [comment, marked as answer]
Hi True_Texan,

We haven't yet sent out more info on how to request a high-detail area to be added on Mars. We may still be able to do this for the next STPro release, depending on if we can fix an internal issue with our planet data processing tools.

Please let us know a latitude/longitude of a place on Mars, or browse the HiRISE data website and find a dataset that you would like us to add.

We already have high fidelity data in a variety of places including Gale Crater, Jezero Crater, some large areas north of Gale Crater, and if I recall correctly, some areas north of Olympus Mons. You can explore our current Mars model to find these.

**Comment by connorjak** [comment]
All teams, you may use this thread or another discussion to request a place on Mars.
